specification/resources/resource-manager/readme.md

```diff
@@ -128,14 +128,14 @@ input-file:
 ```
 
 
-### Tag: package-resources-2021-04
+### Tag: package-resources-2022-09
 
 
-These settings apply only when `--tag=package-resources-2021-04` is specified on the command line.
+These settings apply only when `--tag=package-resources-2022-09` is specified on the command line.
 
-``` yaml $(tag) == 'package-resources-2021-04'
+``` yaml $(tag) == 'package-resources-2022-09'
 input-file:
-- Microsoft.Resources/stable/2021-04-01/resources.json
+- Microsoft.Resources/stable/2022-09-01/resources.json
```

Comment:
please add a new tag instead of modifying existing one

Reply:
Hi Lei, I'm not sure if I've made the correct change. Could you tell me what I'm missing here? Does the order of the tags matter?

Reply:
the order doesn't matter, but don't delete the `package-resources-2021-04` tag, because it may be referenced somewhere.